src/lib/matchers.rs

```diff
@@ -678,9 +677,9 @@ mod tests {
                             "{\n\"src\": 1,\n\"dest\": 3\n}",
                             "{\n\"src\": 0,\n\"dest\": 0\n}",
                             "{\n\"src\": 2,\n\"dest\": 4\n}"]
-                .iter()
-                .map(|s| String::from(*s))
-                .collect::<Vec<String>>();
+            .iter()
```

Comment:
It isn't easy to see from the context of the diff, which I suspect is wrong (the diff I see in gitk is not the same as the one here on GH). This is the new formatting:
```rust
        let expected = vec!["\"matches\": [",
                            "{\n\"src\": 1,\n\"dest\": 3\n}",
                            "{\n\"src\": 0,\n\"dest\": 0\n}",
                            "{\n\"src\": 2,\n\"dest\": 4\n}"]
            .iter()
            .map(|s| String::from(*s))
            .collect::<Vec<String>>();
```
This is the same text from master branch:
```rust
        let expected = vec!["\"matches\": [",
                            "{\n\"src\": 1,\n\"dest\": 3\n}",
                            "{\n\"src\": 0,\n\"dest\": 0\n}",
                            "{\n\"src\": 2,\n\"dest\": 4\n}"]
                .iter()
                .map(|s| String::from(*s))
                .collect::<Vec<String>>();
```